includes/class-fs-logger.php

```diff
@@ -349,14 +368,21 @@ public static function _set_storage_logging( $is_on = true ) {
 KEY `function` (`function` ASC),
 KEY `type` (`type` ASC))" );
 			} else {
-				/**
-				 * Drop logging table.
-				 */
-				$result = $wpdb->query( "DROP TABLE IF EXISTS $table;" );
+                /**
+                 * Since logging table does not exist anymore, we need to turn off all instances
+                 */
+                foreach ( self::$LOGGERS as $logger ) {
```

Comment:
I also think this isn't needed. Because we have just turned off "DB" storage of the log. The actual log happens with the constant `WP_FS__DEBUG_SDK`. So if DB storage is turned off, let there still be memory logging within the `_log` and it will show up in the JavaScript console. The only fix needed is turning off the `$_isStorageLoggingOn`  in line 385 like you did :)